src/components/Autocomplete/Autocomplete.spec.jsx

```diff
@@ -284,7 +284,7 @@ describe('Autocomplete', () => {
 			});
 
 			afterEach(() => {
-				if (wrapper) {
+				if (wrapper && wrapper.exists()) {
```

Comment:
fixes some flaky tests where we were trying to unmount components already unmounted.